tests/testthat/test-raster-data-elements.R

```diff
@@ -0,0 +1,298 @@
+# Tests for spatial raster 
+test_that('add_raster errors when not provided the mandatory input elements', {
+  file_name = "Rasterfiles.zip" 
+  file_description = "A Raster File"
+  attribute_list =  "attribute list"
+  physical = "physical list"
+  spatial_reference = "NAD_1983_StatePlane_California_I_FIPS_0401"
+  horizontal_accuracy = "No Information"
+  vertical_accuracy =  "No Information"
+  cell_size_x = "25"
+  cell_size_y = "25"
+  number_of_bands = "10"
+  raster_origin = "Upper Left"
+  rows = "200"
+  columns = "6"
+  verticals = "1"
+  cell_geometry = "pixel"
+  
+  expect_error(add_raster(file_description = file_description,
```

Comment:
I feel like these tests should be failing because we are not passing in a parent element

Reply:
I agree. I think it still passes the test because the first error it throws is the error I expected but I think it should additionally throw an error because the parent element is missing. I will add that in now.